src/adapter/src/catalog/migrate.rs

```diff
@@ -52,16 +66,26 @@ pub(crate) async fn migrate(catalog: &mut Catalog) -> Result<(), anyhow::Error>
 
     let mut tx = storage.transaction().await?;
     // First, do basic AST -> AST transformations.
-    rewrite_items(&mut tx, |_tx, _stmt| Ok(()))?;
+    // rewrite_items(&mut tx, None, |_tx, _cat, _stmt| Box::pin(async { Ok(()) })).await?;
```

Comment:
Is this meant to be here?

Reply:
Yes because getting the signature right is annoying and in case we need to add a migration while the complex migration is still in the code base, wanted to make that as painless as possible.